package.json

```diff
@@ -16,8 +16,12 @@
     "@codesandbox/sandpack-react": "^2.6.1",
     "@cookbookdev/docsbot": "^4.21.5",
     "@cosmjs/cosmwasm-stargate": "^0.31.0",
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
```

Comment:
I don't see anywhere that you're actually using emotion, is this a peer dependency of mui or something?

Reply:
https://stackoverflow.com/a/69632381
Yes it is.

Reply:
That stackoverflow just tells me that they're using styled components, not that it's a peer dependency.  There shouldn't be any reason to add emotion to our dependencies unless if either:

1. you need to use it directly (in general you shouldn't, we have tailwind for this, the only possible exception would be if you need to pass overrides to a library that internally uses it)
2. it's a peer dependency of a package you depend on (what this means is that the package has it listed in it's `peerDependencies` list in the `package.json`, indicating that the dependency must be installed by the consumer; typically this only happens for things like plugins defining what they are plugging into or things like React where there can only be one copy of the dependency in the build graph)